hips/utils/tile.py

```diff
@@ -0,0 +1,18 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""HEALpy wrapper functions.
+
+This module contains wrapper functions around HEALPix utilizing
+the healpy library.
+"""
+
+from pathlib import Path
+
+__all__ = [
+    'tile_path',
+]
+
+
+def tile_path() -> Path:
```

Comment:
This function doesn't do anything useful yet.
Do you plan to implement something here now?
If no, then I would suggest you remove this function / file for now, and only add it in a future PR if / when you need it.